### PR TITLE
Table formatter

### PR DESCRIPTION
**Pull request by vikrkuma**
adds a table formatter that will add html table tags when ~table is used in the graft

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "graft"
-version = "0.1.0"
+version = "1.0.0"
```
Comment (amitu) [inline Cargo.toml]: Big jump :-p, make it 0.2.0 or something.

**Comment by amitu** [comment]
Lets discuss.

File: Cargo.toml (hunk 1 above)
Comment (vikrkuma) [reply Cargo.toml]: Done

**Comment by vikrkuma** [comment]
> Lets discuss.

Done changes to get output as List of list of strings

**Pull request merged**